docs/stellar-core.1

```diff
@@ -0,0 +1,389 @@
+.\" Automatically generated by Pandoc 1.16.0.2
```

Comment:
I would not bother with this and just have a minimalist file that doesn't need to be maintained